SkycoinProject/skywire-mainnet
# Feature/route finder single route

Fixes https://github.com/skycoin/skywire/issues/567

 Changes:	
Changed route-finder API to accept a list of routes to find

How to test this PR:
With skywire-services under `feature/route-finder-single-route` branch run `make integration-build; make integration-run-generic`. Then `make integration-startup`. Try to use any app, they should be able to find routes from one visor to another.
